src/conditions/check_fields.rs

```diff
@@ -523,7 +524,10 @@ impl CheckFieldsConfig {
 
 #[typetag::serde(name = "check_fields")]
 impl ConditionConfig for CheckFieldsConfig {
-    fn build(&self) -> crate::Result<Box<dyn Condition>> {
+    fn build(
+        &self,
+        _enrichment_tables: &enrichment::TableRegistry,
+    ) -> crate::Result<Box<dyn Condition>> {
```

Comment:
I'd love for this too to just take `None` if you don't have to bother with any enrichment tables, but we can leave that for a follow-up PR.